content/cpp/concepts/vectors/terms/erase/erase.md

```diff
@@ -0,0 +1,97 @@
+---
+Title: '.erase()'
+Description: 'Removes a single element or a range of elements from a vector.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Vectors'
+  - 'Arrays'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **.erase()** function removes a single element or a range of elements from a specific position of the `vector`.
+When an element is remove with *.erase()* method, the size of the `vector` decrease. Therefore the elements after the ones deleted are moved to a new position.
+The position of the elements to remove are identified with `.begin()` and/or `.end()` functions.
+
+## Syntax
+
+```  
+vector.erase(position);  
+vector.erase(starting_position, ending_position);   
+```
+The firsr syntax removes a single element.
+The second syntax removes a range of elements.
```

Comment:
```suggestion
```  
To remove a single element from the vector:

```
vector.erase(position)
```

- `position`: An iterator pointing to the element to be removed.

To eliminate a range of elements:

```
vector.erase(starting_position, ending_position)
```

- `starting_position`: An iterator indicating the beginning of the range to be removed.
- `ending_position`: An iterator indicating the end of the range to be removed.
```

Reply:
Modifications:
1. The above format maintains a structure for the reader.
2. Also, whenever we give a syntax, make sure to describe the parameters in short as above.